main.go

```diff
@@ -43,51 +44,51 @@ const (
 	initOnly                      = flag.Bool("init-only", false, "Exit after initialize, generate self keys. Need for auto-test environment.")
 	keepAliveModeS                = flag.String("keepalive", KEEPALIVE_TRANSPARENT_STRING, KEEPALIVE_TRANSPARENT_STRING+" - keepalive from user to server if both support else doesn't keepalive. "+KEEPALIVE_NO_BACKEND_STRING+" - force doesn't use keepalive connection to backend, but can handle keepalive from user.")
-	keepAliveCustomerTimeout      = flag.Duration("keepalive-customer-timeout", time.Minute*15, "When keepalive in mode '"+KEEPALIVE_NO_BACKEND_STRING+"' - how long time keep customer's connection. In '"+KEEPALIVE_TRANSPARENT_STRING+"' mode timeout don't used and both connection close when one of backend or customer close self connection.")
+	keepAliveCustomerTimeout      = flag.Duration("keepalive-customer-timeout", time.Minute*15, "When keepalive in mode '"+KEEPALIVE_NO_BACKEND_STRING+"' - how long should the connection be maintained. In '"+KEEPALIVE_TRANSPARENT_STRING+"' mode, `timeout` isn't used and both connections close when either the backend or customer close self connection.")
 	logLevel                      = flag.String("loglevel", "warning", "fatal|error|warning|info|debug")
 	logOutput                     = flag.String("logout", "-", "Path to logout. Special: '-' (without quotes) - stderr")
 	logrotateMaxAge               = flag.Int("logrotate-age", 30, "How many days keep old backups")
-	logrotateMaxCount             = flag.Int("logrotate-count", 30, "How many old backups keep. 0 mean infinite")
-	logrotateMb                   = flag.Int("logrotate-mb", 100, "logrotate by size in megabytes. 0 Mean no logrotate by size.")
-	logrotateTime                 = flag.String("logrotate-time", "", "minutely|hourly|daily|weekly|monthly|yearly|\"\", empty or none mean no logrotate by time. Weekly - rotate log at midnight from sunday to monday")
-	minTLSVersion                 = flag.String("min-tls", "", "Minimul supported tls version: ssl3,tls10,tls11,tls12. Default is golang's default.")
-	noLogStderr                   = flag.Bool("no-log-stderr", false, "supress log to stderr")
-	nonCertDomains                = flag.String("non-cert-domains", "", "No obtain certificate for mathed domains. Regexpes separated by comma.")
-	pidFilePath                   = flag.String("pid-file", "lets-proxy.pid", "Write pid of process. When used --daemon - lock the file for prevent double-start daemon.")
-	preventIDNDecode              = flag.Bool("prevent-idn-decode", false, "Default domain show in log as 'domain.com' or 'xn--d1acufc.xn--p1ai' ('домен.рф'). When option used it will shown as 'domain.com' or 'xn--d1acufc.xn--p1ai', without decode idn domains.")
+	logrotateMaxCount             = flag.Int("logrotate-count", 30, "How many old backups to keep. 0 for keep indefinitely.")
```

Comment:
I think abount word infinite or unlimited.
I saw indefinitely dictionaris and examples in godoc: indefinitely more like to undefined, then unlimited.